src/test/java/net/wasdev/wlp/ant/SpringBootUtilTest.java

```diff
@@ -0,0 +1,133 @@
+package net.wasdev.wlp.ant;
+
+import java.io.File;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.tools.ant.BuildException;
+import org.apache.tools.ant.Project;
+import static org.junit.Assert.*;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import junit.framework.Assert;
+import net.wasdev.wlp.ant.SpringBootUtilTask;
+import net.wasdev.wlp.ant.install.InstallLibertyTask;
+
+public class SpringBootUtilTest {
+	
+	static final Boolean isWindows;
```

Comment:
Use the OSUtil here: https://github.com/WASdev/ci.common/blob/master/src/main/java/net/wasdev/wlp/common/plugins/util/OSUtil.java

Reply:
done